packages/data-service-generator/src/models.ts

```diff
@@ -645,7 +645,8 @@ export enum EnumActionStepStatus {
 
 export enum EnumAuthProviderType {
   Http = 'Http',
-  Jwt = 'Jwt'
+  Jwt = 'Jwt',
+  Auth0 = 'Auth0'
```

Comment:
@Shurtu-gal I believe that if you changed `models.ts`, you should generate and commit *all* `models.ts` files for all packages (there are 6 of them IIRC, you just committed two of them in this PR)